src/components/InRequest/InRequestEditPanel.tsx

```diff
@@ -91,21 +91,14 @@ export const InRequestEditPanel: FunctionComponent<IInRequestEditPanel> = (
   }, [eta]);
 
   const onOpen = () => {
-    const transRes = {
-      ...props.data,
-      hasExistingCAC: props.data?.hasExistingCAC ? "yes" : "no",
-      isNewCivMil: props.data?.isNewCivMil ? "yes" : "no",
-      isNewToBaseAndCenter: props.data?.isNewToBaseAndCenter ? "yes" : "no",
-      isTraveler: props.data?.isTraveler ? "yes" : "no",
-      isEmpNotInGAL: props.data?.employee ? false : true,
```

Comment:
Technically this one is still needed -- but will be OBE when I remove the Checkbox in my GUI update.